### PR TITLE
V0.8.0 rc

### PR DESCRIPTION
**Pull request by rhornung67**
Starting a release candidate branch for reviewer feedback.

Most of the changes have been reviewed and merged into develop.

For this PR review, please focus on newer commits updating/adding user docs, release notes, etc.

### CODE REVIEW

File: scripts/lc-builds/blueos-EA_nvcc9_clang6.0.0.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+i#!/usr/bin/env bash
```
Comment (davidbeckingsale) [inline scripts/lc-builds/blueos-EA_nvcc9_clang6.0.0.sh]: Typo

Comment (rhornung67) [reply scripts/lc-builds/blueos-EA_nvcc9_clang6.0.0.sh]: Fixed. God catch! We really need to get our CI set up on the LC....

Comment (davidbeckingsale) [reply scripts/lc-builds/blueos-EA_nvcc9_clang6.0.0.sh]: We are making some progress... 😄

**Comment by rhornung67** [comment]
@davidbeckingsale @artv3 @rchen20 @ajkunen @MrBurmark This is good to go for the release. Can we get some eyeballs to review this so we can get the release out tomorrow. Please review the release notes and other documentation updates. All other changes have been reviewed and merged into develop.  Thanks. --Rich

**Pull request merged**